containers/runtime/driver/run.sh

```diff
@@ -24,7 +24,8 @@ fi
 
 /src/code/bazel-bin/test/cpp/qps/qps_json_driver --quit=true
 
-if [ -n "${BQ_RESULT_TABLE}" ] && [ -r "${METADATA_OUTPUT_FILE}" ]; then
+if [ -n "${BQ_RESULT_TABLE}" ] && [ -r "${METADATA_OUTPUT_FILE}" ] && [ -r "${NODE_INFO_OUTPUT_FILE}" ]; then
```

Comment:
Reading more carefully, I don't think this logic is correct.

I think it should be something like this:

```shell
if [-n "${BQ_RESULT_TABLE}"]; then
  [ -r "${METADATA_OUTPUT_FILE}" ] && cp "${METADATA_OUTPUT_FILE}" metadata.json
  [ -r "${NODE_INFO_OUTPUT_FILE}" ] && cp "${NODE_INFO_OUTPUT_FILE}" node_info.json
  python3 /src/code/tools/run_tests/performance/bq_upload_result.py --bq_result_table="${BQ_RESULT_TABLE}"
fi
```

Unless we are assuming that the metadata and node info files must always be created, but even in that case I think the data should be saved to BigQuery even if the metadata or node_info files are missing.

Reply:
done